xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -1302,10 +1302,11 @@
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="TAlternateKey">To be added.</typeparam>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TAlternateKey">The alternate type of a key for performing lookups.</typeparam>
+        <summary>Gets an instance of a type that may be used to perform operations on the current <see cref="T:System.Collections.Generic.Dictionary`2"/> using a <typeparamref name="TAlternateKey"/> as a key instead of a <typeparamref name="TKey"/>.</summary>
```

Comment:
```suggestion
        <summary>Gets an instance of a type that can be used to perform operations on the current <see cref="T:System.Collections.Generic.Dictionary`2"/> using a <typeparamref name="TAlternateKey"/> as a key instead of a <typeparamref name="TKey"/>.</summary>
```